imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+
+from django.db import models
+
+from django.utils.encoding import python_2_unicode_compatible
+
+PHOTOGRAPHY_TYPES = (
+    ('ad', 'advertising'),
+    ('am', 'amateur'),
+    ('bl', 'black and white'),
+    ('co', 'color'),
+    ('co', 'commercial'),
+    ('cr', 'crime scene'),
+    ('ed', 'editorial'),
+    ('fa', 'fashion'),
+    ('fo', 'food'),
+    ('la', 'landscape'),
+    ('ph', 'photojournalism'),
+    ('po', 'portrait and wedding'),
+    ('st', 'still life'),
+    ('wi', 'wildlife'),
+)
+
+REGIONS = (
+    ('af', 'Africa'),
+    ('an', 'Antarctica'),
+    ('as', 'Asia'),
+    ('au', 'Australia'),
+    ('eu', 'Europe'),
+    ('no', 'North America'),
+    ('so', 'South America'),
+)
+
+
+class ImagerProfileManager(models.Manager):
+    def get_queryset(self):
+        """Return set of all active users."""
+        profiles = super(ImagerProfileManager, self).get_queryset()
+        return profiles.filter(user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """Profile that stands in front of django's User model."""
+    user = models.OneToOneField(settings.AUTH_USER_MODEL,
+                                related_name='profile',
+                                null=False)
+    camera = models.CharField(max_length=20, default='')
+    photography_type = models.CharField(max_length=2,
+                                        choices=PHOTOGRAPHY_TYPES,
+                                        default='amateur')
+    region = models.CharField(max_length=2, choices=REGIONS)
+
+    def __str__(self):
+        return self.user.username
+
+    # order is important
+    objects = models.Manager()  # return all user profiles
```

Comment:
Why do you override the default objects manager that django models give you? Additionally, why do you override the defaults objects manager with just the default manager?